modules/CmpdRegBulkLoader/src/server/routes/CmpdRegBulkLoaderRoutes.coffee

```diff
@@ -338,3 +343,103 @@ exports.purgeFile = (req, resp) ->
 				console.log response
 				resp.end JSON.stringify "Error"
 		)
+
+checkStatus = (response) ->
```

Comment:
This is a nice little utility function. @brianbolt do you happen to know if we have something similar elsewhere in ACAS node routes?